src/Illuminate/Foundation/Console/TinkerCommand.php

```diff
@@ -1,10 +1,19 @@
 <?php namespace Illuminate\Foundation\Console;
 
-use Psy\Shell;
 use Illuminate\Console\Command;
+use Illuminate\Foundation\Console\Tinker\Presenters\EloquentCollectionPresenter;
+use Illuminate\Foundation\Console\Tinker\Presenters\EloquentModelPresenter;
+use Illuminate\Foundation\Console\Tinker\Presenters\IlluminateApplicationPresenter;
+use Psy\Configuration;
+use Psy\Shell;
+use Symfony\Component\Console\Input\InputArgument;
 
```

Comment:
I think the imported classes are usually sorted by length instead of alphabetically in Laravel.